main.go

```diff
@@ -179,9 +181,14 @@ func main() { //nolint:funlen,maintidx
 
 	secretCache := createSecretCacheConfig(platform)
 	deploymentCache := createDeploymentCacheConfig(platform)
+
+	uingress := &unstructured.Unstructured{}
+	uingress.SetGroupVersionKind(gvk.OpenshiftIngress)
+
 	cacheOptions := cache.Options{
 		Scheme: scheme,
 		ByObject: map[client.Object]cache.ByObject{
+			uingress: {},
```

Comment:
Not sure about the impact of this, since there's no specific configuration, then I think this line does not have effects on the cache, am I wrong ?

Reply:
yep, i forgot to put the selector on it for "cluster" instance